docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -185,6 +189,12 @@ below shows all its proper subtransactions on the right (yellow boxes).
    :align: center
    :width: 100%
 
+To illustrate contract keys, suppose that the contract key for a `PaintOffer` consists of the reference number and the painter as the key maintainer.
+So Alice can refer to the `PaintOffer` by its key `(P, P123)`.
```

Comment:
Is this deliberately using single backticks rather than double?  I guess that is what we do elsewhere in this section, though the rest of the docs would use double backticks (and get code formatting). But probably too much effort to update this entire section.

Reply:
It's consistent with how it is done in this section, so I'll leave it for now. If you think that it's important to change everything to double backticks, then let's do that as part of another ticket.